corehq/apps/reports/standard/__init__.py

```diff
@@ -41,6 +41,8 @@ class CustomProjectReport(ProjectReport):
     dispatcher = CustomProjectReportDispatcher
     emailable = True
     is_public = False
+    is_support_translations = False
+    languages = []
```

Comment:
For "default" values (either in a function keyword argument or in a class body like here) it's best not to use mutable values, because if any instance of the class or instance of a subclass changes it by mutating the object, that change affects all other instances. In this case, I think you can just use `language = None`.

Reply:
+1. Mutable default values can sometimes result in pretty [gnarly bugs](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).